run.py

```diff
@@ -0,0 +1,14 @@
+#!flask/bin/python
+import sys
+from app import mulungwishi_app
+
+# This line is important so flake8 must ignore this one. 
```

Comment:
yes but what does flake8 need to ignore? you're not explaining this for someone that neither sees the error, nor knows what they can and *cannot* do with this line.